src/pykiso/test_coordinator/test_execution.py

```diff
@@ -323,6 +324,23 @@ def collect_test_suites(
     return list_of_test_suites
 
 
+def abort(reason: str = None) -> None:
+    """Quit ITF test and log an error if a reason is indicated and
+    if any errors occurred it logs them.
+
+    :param reason: reason to abort, defaults to None
+    """
+    if reason:
+        log.critical(reason)
+    log.exception("Tests were aborted because of the following error :")
+    log.error(
+        "Non recoverable error occurred in communication, aborting entire test execution."
```

Comment:
Shall this be a generic error message ? 
"occurred in communication" confuses me

Reply:
Yes it can be more generic since the function can be used everywhere, this error in communication is because I implemented the function for an error in a cc_receive.
Should make it more generic ?

Reply:
Done

Reply:
Done

Reply:
I'd go for something like `log.critical("Non recoverable error occurred during test execution, aborting test session.")`